products/angular-js.md

```diff
@@ -0,0 +1,137 @@
+    extendedSupport: true
+    latest: "1.5.17"
+    latestReleaseDate: 2023-04-14
+    lts: false
+    discontinued: false
+    link: https://xlts.dev/blog/2023-02-07-xlts-for-angularjs-1-5-x-1-5-17?utm_source=endoflife.date&utm_medium=open-source-pr&utm_campaign=xlts-angularjs-blog&utm_content=pr-link
+
+-   releaseCycle: "1.4"
+    releaseDate: 2015-04-13
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.4.14"
+    latestReleaseDate: 2016-10-11
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.4.14
+
+-   releaseCycle: "1.3"
+    releaseDate: 2014-10-13
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.3.20"
+    latestReleaseDate: 2015-09-29
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.3.20
+
+-   releaseCycle: "1.2"
+    releaseDate: 2013-11-06
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.2.32"
+    latestReleaseDate: 2016-10-11
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.2.32
+
+-   releaseCycle: "1.1"
+    releaseDate: 2012-08-13
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.1.5"
+    latestReleaseDate: 2013-05-22
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.1.5
+
+-   releaseCycle: "1.0"
+    releaseDate: 2010-10-20
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.0.8"
+    latestReleaseDate: 2013-08-22
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.0.8
+
+---
+
+> AngularJS is a free and open-source JavaScript-based web framework for developing single-page applications. It aims to simplify both the development and the testing of such applications by providing a framework for client-side model–view–controller (MVC) and model–view–viewmodel (MVVM) architectures, along with components commonly used in web applications and progressive web applications. AngularJS, referred to as Angular 1, was built and maintained by Google.
+
+AngularJS was [deprecated](https://docs.angularjs.org/misc/version-support-status) on December 31, 2021 after a [LTS period](https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c) on the final version 1.8.3 which was released April 7, 2022.
+
+## Extended Long Term Support
+
+The independent team at XLTS.dev offers [extended long term support for AngularJS](https://xlts.dev/angularjs?utm_source=endoflife.date&utm_medium=open-source-pr&utm_campaign=xlts-angularjs-blog&utm_content=pr-link). On January 1st 2022, the XLTS.dev team began maintaining, supporting and releasing a fork of the latest version of AngularJS and provides software updates that fit any of the following criteria:
```

Comment:
I am not against documenting ways to obtain extended support even if it's provided by another company (e.g. not Google in this case). But I feel this section is a little bit too much and should be shortened to a single paragraph like we have done for other products on endoflife.date (such as https://endoflife.date/apache-kafka).

I am also not in favor in putting tracking tags (`utm_xxx`) in links.

@captn3m0 what is your stance about this?

Reply:
Solved on [7e1b3ae](https://github.com/endoflife-date/endoflife.date/commit/7e1b3ae83176086f549ffcaa2b52778cdec7bdf8)